### PR TITLE
Basic usage docs

### PR DESCRIPTION
**Pull request by stringfellow**
@elsmorian @ezissou

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
+# xylem
+
+Library to interact with CarbonCulture's datastore
+
+
+## Installation
+
+Installation using pip::
+
+    pip install https://github.com/CarbonCulture/xylem/archive/v0.4.11.zip
```
Comment (elsmorian) [inline README.md]: I'm not sure why GitHub doesn't give us a "latest" link or button for xylem on https://github.com/CarbonCulture/xylem/releases, as it does for other repos.. might be nice to have this if its a small fix
?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+
+...
+
+```
+
+There's quite a lot of info in this result, but the pertinent points are the
+result keys:
+
+```
+In [5]: _4.keys()
```
Comment (elsmorian) [inline README.md]: might be slightly clearer to manually assign the result to a variable and call that, just for clarity?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+
+https://platform.carbonculture.net/communities/uk-parliament/2/
+
+Cardiff Castle's ID is 947:
+
+https://platform.carbonculture.net/places/cardiff-castle/947/
+
+Note that a Place or Meter may have the same ID as a Community, but Places
+and Meters never share IDs (this is because internally Places and Meters are
+both represented as the same data structure - a physical 'Asset').
```
Comment (elsmorian) [inline README.md]: Maybe bold this up a little, its possibly helpful for understanding internals?

File: README.md (hunk 1 above)
Comment (stringfellow) [inline README.md]: Yeah, I think we just have to keep this updated in line with CHANGES and the version numbers

**Comment by elsmorian** [comment]
This reads nicely, i think it feels more like a tutorial rather than docs, but i think thats more helpful. Nice! :+1:

File: README.md (hunk 1 above)
Comment (elsmorian) [inline README.md]: I had a little skim, couldn't find any info, but lots of other repos have it. Not worth a proper look into now I guess, as you say.

**Pull request merged**